include/openthread/thread.h

```diff
@@ -382,14 +382,27 @@ otError otThreadSetLinkMode(otInstance *aInstance, otLinkModeConfig aConfig);
 /**
  * Get the Thread Network Key.
  *
+ * @param[in]   aInstance     A pointer to an OpenThread instance.
+ * @param[out]  aNetworkKey   A pointer to an `otNetworkkey` to return the Thread Network Key.
+ *
+ * @sa otThreadSetNetworkKey
+ *
+ */
+void otThreadGetNetworkKey(otInstance *aInstance, otNetworkKey *aNetworkKey);
```

Comment:
I think for some security implementation this function might return an error like "forbidden". I think it would be better to return otError to allow such cases in the future.

We can also change return type later, when anyone provides such security implementation.